src/commands/service/get-display-name.ts

```diff
@@ -0,0 +1,72 @@
+import commander from "commander";
+import { read as readBedrockYaml } from "../../lib/bedrockYaml";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { logger } from "../../logger";
+import decorator from "./get-display-name.decorator.json";
+import { build as buildError, log as logError } from "../../lib/errorBuilder";
+import { errorStatusCode } from "../../lib/errorStatusCode";
+
+export interface CommandOptions {
+  path: string | undefined;
+}
+
+/**
+ * Executes the command, can all exit function with 0 or 1
+ * when command completed successfully or failed respectively.
+ *
+ * @param opts validated option values
+ * @param exitFn exit function
+ */
+export const execute = async (
+  opts: CommandOptions,
+  exitFn: (status: number) => Promise<void>
+): Promise<void> => {
+  // The assumption is that this command should be ran from the directory where bedrock.yaml exists
+  try {
+    if (!opts.path) {
+      throw buildError(
+        errorStatusCode.CMD_EXE_ERR,
+        "service-get-display-name-path-missing-param-err"
+      );
+    }
+    const bedrockFile = readBedrockYaml(process.cwd());
```

Comment:
why `process.cwd()`? what happens to `opts.path`?

Reply:
We're currently requiring that `service` and `project` commands be run only at the root of an application repo, so it's expected that there is a `bedrock.yaml` file in the current directory.

Reply:
@dennisseah `opts.path` is not for where bedrock.yaml is, it's used for finding a corresponding display name for a service that is at `opts.path`

Reply:
got it. thanks